### PR TITLE
Added Event Registrants Tab under Event Management Dashboard

### LINKED ISSUE #1750: ADMIN REDESIGN: redesign the Event Management's Registrant tab for the talawa Admin portal
**Is your feature request related to a problem? Please describe.**
we are planning redesigning the admin portal, this issue is a part of that redesigning flow.

**Describe the solution you'd like**
we want to redesign the Event Management Registrant Tab for the talawa Admin portal
- we want to club show registrant and check-ins registrant in a single parent tab i.e registrant
- The show registrant is ready for now please leave check-in registrant button non functional for now.
- After clicking on the add registrant button , the registrant should dissapper from this screen after a react toast with success message.

**Screenshots**
- Attaching the screenshots for reference.

Current Situation:
Screenshot 2024-03-20 at 12 22 18 AM

Desired Result:
Screenshot 2024-03-20 at 12 17 14 AM

Related Issues: 

#1748

Please refer the figma file for a detailed understanding : 
https://www.figma.com/file/WksD4uIa5sQJMIk3pR4l0c/Talawa-Design-System-(Official)?type=design&node-id=3447%3A59&mode=design&t=fkgMPaaz67qLwmMz-1

**Potential internship candidates**
Please read this if you are planning to apply for a Palisadoes Foundation internship https://github.com/PalisadoesFoundation/talawa/issues/359

### PR DESCRIPTION
**Pull request by syedali237**
**What kind of change does this PR introduce?**

Feature

**Issue Number:**

Fixes #1750 

**Did you add tests for your changes?**

Yes

**Snapshots/Videos:**

https://github.com/user-attachments/assets/af20d577-93e6-43ed-ad3e-33e241053c4b

**If relevant, did you update the documentation?**

No

**Summary**

Added the event registrants tab where admin can check in and add registrants of an event.

**Does this PR introduce a breaking change?**

No

**Other information**

Added a new package : @pdfme/common and changed its template schema according to the package. (This might give a formatting/linting error).

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-admin/blob/master/CONTRIBUTING.md)?**

Yes

## Summary by CodeRabbit

- **New Features**
	- Introduced a new component for managing and displaying event registrant information.
	- Added a GraphQL query to retrieve event attendees based on event ID.
	- Enhanced localization support for event registrants in multiple languages (English, French, Hindi, Spanish, Chinese).

- **Bug Fixes**
	- Updated button labels for clarity in user interactions.

- **Documentation**
	- Added mock data for testing event registration functionalities.

- **Chores**
	- Updated ESLint configuration to ignore additional file types.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,6 +20,7 @@
     "@mui/x-charts": "^7.22.2",
     "@mui/x-data-grid": "^7.22.1",
     "@mui/x-date-pickers": "^7.18.0",
+    "@pdfme/common": "^5.2.11",
```
[the remark this answers is not in the document]
Comment (syedali237) [reply package.json:23]: Tested the files where these packages work, they are compatible with each other. In the future we can consider upgrading the @pdfme/generator & @pdfme/schemas packages to a higher version.

**Comment by syedali237** [comment]
@duplixx Is .eslintrc.json allowed to make changes?  just added "**/*.svg" in the file.

Screenshot 2024-12-25 at 12 57 21 AM

**Comment by syedali237** [comment]
Should I raise an issue regarding the test coverage of this component as it would also require to be migrated into vitest.

**Comment by palisadoes** [comment]
Please fix the test coverage in this PR

**Comment by syedali237** [comment]
@palisadoes increased the test coverage.

**Review by palisadoes — CHANGES_REQUESTED** [review]
Please move all your CSS to this stylesheet: 
- https://github.com/PalisadoesFoundation/talawa-admin/tree/develop-postgres/src/style

It will help us implement a light/dark mode in future, and possibly centralized overrides

**Comment by syedali237** [comment]
@palisadoes, I am already using all the CSS properties from that file only. The CSS file for this component is only present if in case an individual file is required  in the future.

**Comment by palisadoes** [comment]
In that case please delete all the CSS files in the directories where the files you have edited reside. 

Make sure there are no references to the local CSS files

**Review by palisadoes — APPROVED** [review]

**Pull request merged**